app/src/main/java/com/vatsal/kesarwani/therapy/Encryption/Base64.java

```diff
@@ -0,0 +1,660 @@
+package com.vatsal.kesarwani.therapy.Encryption;
+
+import java.io.UnsupportedEncodingException;
+
+/**
+ * Utilities for encoding and decoding the Base64 representation of
```

Comment:
Why imported this class to the project manually and not used via dependencies?

Reply:
I had to make some changes in the 'ENCODE' and 'DECODE' sections of this class, as it was for WEB_ENCODE only
and was showing "\m" string in last of every encrypted string